reasoner/processor/AbstractProcessor.java

```diff
@@ -188,6 +190,10 @@ public Driver<Monitor> monitor() {
         public ReasonerPerfCounters perfCounters() {
             return perfCounters;
         }
+
+        public boolean explainEnabled() {
+            return explainEnabled;
+        }
```

Comment:
Since we're factoring the conjunction into small bits for re-usable answers, we break the present approach to explanations. We now use this as a switch to turn off reasoner optimisations which break explanations.